addons/l10n_kr/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name': "Korea",
+    'icon': '/account/static/description/l10n.png',
+    'countries': ['kr'],
+    'version': '1.0',
+    'category': 'Localization',
+    'description': """
+This module changes the address format to Korean format.
+    """,
+    'depends': [
+        'base',
+        'account',
+    ],
```

Comment:
account already depends on base indirectly, so no need to precise it here

Reply:
Additionally, you can add the 

```py
    'auto_install': ['account'],
```

here so that the module get auto installed when a new db is created with a Koran company